src/lib.rs

```diff
@@ -161,6 +161,14 @@ impl<T: Clone + Signed> Complex<T> {
 
 #[cfg(any(feature = "std", feature = "libm"))]
 impl<T: Float> Complex<T> {
+    /// Create a new Complex with a given phase -- `exp(i * phase)`
+    ///
+    /// [cis (mathematics)]: https://en.wikipedia.org/wiki/Cis_(mathematics)
```

Comment:
This link is useful context for the name, but you're not actually using it in markdown, so it won't show in the documentation.